app-backend/batch/src/main/scala/com/azavea/rf/batch/landsat8/ImportLandsat8C1.scala

```diff
@@ -186,80 +172,81 @@ case class ImportLandsat8C1(startDate: LocalDate = LocalDate.now(ZoneOffset.UTC)
+              metadataFiles = List(),
+              bands = List(band)
+            )
         }
-        ).map {
-        case (resolution, tiffPath, band) =>
-          Image.Banded(
-            rawDataBytes = 0,
-            visibility = Visibility.Public,
-            filename = tiffPath,
-            sourceUri = s"${getLandsatUrl(productId)}/${tiffPath}",
-            owner = Some(systemUser),
-            scene = sceneId,
-            imageMetadata = Json.Null,
-            resolutionMeters = resolution,
-            metadataFiles = List(),
-            bands = List(band)
-          )
-      }
 
-      val scene = Scene.Create(
-        id = Some(sceneId),
-        visibility = Visibility.Public,
-        tags = tags,
-        datasource = landsat8Config.datasourceUUID,
-        sceneMetadata = sceneMetadata.asJson,
-        name = landsatPath,
-        owner = Some(systemUser),
-        tileFootprint = tileFootprint,
-        dataFootprint = dataFootprint,
-        metadataFiles = List(s"${landsat8Config.awsLandsatBaseC1}${landsatPath}/${productId}_MTL.txt"),
-        images = images,
-        thumbnails = createThumbnails(sceneId, productId),
-        ingestLocation = None,
-        filterFields = SceneFilterFields(
-          cloudCover = cloudCover,
-          sunAzimuth = sunAzimuth,
-          sunElevation = sunElevation,
-          acquisitionDate = acquisitionDate
-        ),
-        statusFields = SceneStatusFields(
-          thumbnailStatus = JobStatus.Success,
-          boundaryStatus = JobStatus.Success,
-          ingestStatus = IngestStatus.NotIngested
-        ),
-        sceneType = Some(SceneType.Avro)
-      )
-      Some(scene)
+        val scene = Scene.Create(
+          id = Some(sceneId),
+          visibility = Visibility.Public,
+          tags = tags,
+          datasource = landsat8Config.datasourceUUID,
+          sceneMetadata = sceneMetadata.asJson,
+          name = landsatPath,
+          owner = Some(systemUser),
+          tileFootprint = tileFootprint,
+          dataFootprint = dataFootprint,
+          metadataFiles = List(s"${landsat8Config.awsLandsatBaseC1}${landsatPath}/${productId}_MTL.txt"),
+          images = images,
+          thumbnails = createThumbnails(sceneId, productId),
+          ingestLocation = None,
+          filterFields = SceneFilterFields(
+            cloudCover = cloudCover,
+            sunAzimuth = sunAzimuth,
+            sunElevation = sunElevation,
+            acquisitionDate = acquisitionDate
+          ),
+          statusFields = SceneStatusFields(
+            thumbnailStatus = JobStatus.Success,
+            boundaryStatus = JobStatus.Success,
+            ingestStatus = IngestStatus.NotIngested
+          ),
+          sceneType = Some(SceneType.Avro)
+        )
+        Some(scene)
       }
     }
   }
 
-  @SuppressWarnings(Array("TraversableHead"))
+  @SuppressWarnings(Array("ListAppend", "TraversableHead", "TraversableLast"))
```

Comment:
Seems like the `ListAppend` isn't a big deal because the list is of known size. `TraversableLast` to go along with `TraversableHead`.

Reply:
Yeah, that was my idea here - the most we have are 4 points